File: JAVA/BasicCalculator/app/src/main/java/com/sumeyyebuyukguclu/ckoguzelkalkulator/MainActivity.java
package com.sumeyyebuyukguclu.ckoguzelkalkulator;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    EditText number1;
    EditText number2;
    TextView result;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        number1=findViewById(R.id.editTextTextPersonName);
        number2=findViewById(R.id.editTextTextPersonName2);
        result=findViewById(R.id.textView);
    }
    public void topla(View view)
    {
        int numb1 = Integer.parseInt(number1.getText().toString());
        int numb2 = Integer.parseInt(number2.getText().toString());

        int res =  numb1 + numb2;

        result.setText(String.valueOf(res));
    }

    public void çıkar(View view)
    {
        int numb1 = Integer.parseInt(number1.getText().toString());
        int numb2 = Integer.parseInt(number2.getText().toString());

        int res =  numb1 - numb2;

        result.setText(String.valueOf(res));
    }
}

File: MainActivity.kt
package com.sumeyyebuyukguclu.myapplication

import android.annotation.SuppressLint
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    @SuppressLint("SetTextI18n")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val TextViewID = findViewById<TextView>(R.id.TextViewID)


        TextViewID.text = """
            That is something that I can not deny
            You put my soul from worst to best
            That’s why I value my dear lover.

            You just don’t know what you have done for me
            You even pushed me to the best that I can be
            You really are an angel sent from above
            To take care of me and shower with love
        """.trimIndent()
    }
}